### PR TITLE
models: add note about the store injection

### PR DESCRIPTION
**Pull request by knownasilya**
Basically that it's automatically available on routes.

Not sure if this is the best place for it. Was looking for like a blockquote example, but didn't find one, so maybe there is some syntax to make this stand out a bit.

### CODE REVIEW

File: guides/v3.7.0/models/index.md [hunk 1]
```diff
@@ -182,6 +182,8 @@ for your app's models. Both routes and their corresponding controllers have acce
 this shared store; when they need to display or modify a model, they
 first ask the store for it.
 
+Note: The Ember Data store is automatically injected on every Route instance as `this.store`.
```
Comment (locks) [inline guides/v3.7.0/models/index.md]:
```suggestion

<div class="cta">
  <div class="cta-note">
    <div class="cta-note-body">
      <div class="cta-note-heading">Zoey says...</div>
      <div class="cta-note-message">
        Ember Data injects the store service in every route and controller,
        so you can access it as `this.store`!
      </div>
    </div>
    <img src="/images/mascots/zoey.png" role="presentation" alt="Ember Mascot">
  </div>
</div>
```

I got the markup from Chris 😁 I made a tweak to the prose, what do you think?

Comment (knownasilya) [reply guides/v3.7.0/models/index.md]: Wait, controllers get the store too?

Comment (locks) [reply guides/v3.7.0/models/index.md]: I believe so: https://github.com/emberjs/data/blob/master/addon/setup-container.js#L68

**Comment by knownasilya** [comment]
@locks

**Pull request merged**